com.claudiocambra.brickbuster.json

```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "com.claudiocambra.brickbuster",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "brickbuster_linux.x86_64",
+    "modules": [
+        {
+            "name": "BrickBuster",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D brickbuster_linux.x86_64 /app/bin/brickbuster_linux.x86_64"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "path": "brickbuster_linux.x86_64"
```

Comment:
You need to download it, so it should be a "url".

This won't work.

Reply:
Right, I've changed this to a URL now:

```
            "sources": [
                {
                    "type": "file",
                    "path": "brickbuster_linux.x86_64"
                    "path": "https://github.com/claucambra/BrickBuster/releases/download/1.0/brickbuster_linux.x86_64",
                    "tag": "1.0"
                }
            ]
```